samples/net/openthread/coap_utils/coap_client_utils.h

```diff
@@ -0,0 +1,12 @@
+#ifndef __COAP_CLIENT_UTILS_H__
+#define __COAP_CLIENT_UTILS_H__
+
+void coap_client_utils_init(void);
```

Comment:
as this part was moved to a separate module consider doxygen description for coap_client_utils